bin/dh_virtualenv

```diff
@@ -65,6 +65,10 @@ def main():
         dh = DebHelper(packages=options.package or None)
     else:
         dh = DebHelper(options)
+    if hasattr(sys, 'real_prefix'):
+        log.error('Already in a virtualenv. This is likely to generate an '
+                  'invalid package (run "deactivate" first?)')
+        return 2
```

Comment:
We should return 1 here instead. 2 is quite often indicating wrongly crafted command, bad use of built-ins etc. Feel free to convince otherwise!

Reply:
(Done!)